.github/ISSUE_TEMPLATE/documentation.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Documentation
+about: 'Please provide your feedback here instead: https://github.com/prisma/prisma-content-feedback/issues/new?template=documentation-feedback.md.'
+
```

Comment:
I weren't sure if this will be part of the current prisma or it will have separate docs. 
I would say until documentation is not seeded somewhere else entire document can be removed ?